velox/docs/monthly-updates/november-2023.rst

```diff
@@ -0,0 +1,102 @@
+********************
+November 2023 Update
+********************
+
+Core Library
+============
+
+* Add support to export constant vector as `Arrow REE
+  array <https://arrow.apache.org/docs/format/Columnar.html#run-end-encoded-layout>`_. :pr:`7327`, :pr:`7398`
+* Add spilling support for distinct aggregation. :pr:`7305`
+* Add spilling support for aggregations over sorted inputs. :pr:`7526`
+* Add support to lazily create the spill directory. :pr:`7660`
+* Add DirectBufferedInput: a selective BufferedInput without caching. :pr:`7217`
+* Add config ``merge_exchange.max_buffer_size`` to limit the total memory used by exchange clients. :pr:`7410`
+* Add configs ``sort_writer_max_output_rows`` and ``sort_writer_max_output_bytes`` to limit memory usage of sort writer. :pr:`7339`
+* Add termination time to TaskStats. :pr:`7479`
```

Comment:
Let's ask @kevinwilfong to help phrase in terms of user-visible improvement.

Reply:
I know this one from the Presto 285 release notes.
`Add termination time to TaskStats. This is the time when the downstream workers finish consuming results. Clients such as
  Prestissimo can use this metric to clean up tasks.`

Reply:
Technically it can be the time when the workers finish consuming results, or the task fails or is cancelled or aborted.

If you want to make it a little less Presto specific.

> Add termination time to TaskStats. This is the time when the Task enters its terminal state and begins freeing all resources held by the Drivers. It can be used to answer the question, when did the Task finish execution *and* serving results.